src/arns.lua

```diff
@@ -445,14 +538,26 @@ function arns.pruneRecords(currentTimestamp)
 	return prunedRecords
 end
 
+-- prune auctions that have expired
+function arns.pruneAuctions(currentTimestamp)
+	local prunedAuctions = {}
+	for name, auction in pairs(arns.getAuctions()) do
+		if auction.endTimestamp <= currentTimestamp then
```

Comment:
Note here to make sure that end timestamp is handled consistently in various functions (inclusively or exclusively during bidding, pruning, etc.)